Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -582,10 +582,16 @@ Vector<RefPtr<StyleRuleBase>> CSSParserImpl::consumeRegularRuleList(CSSParserTok
     Vector<RefPtr<StyleRuleBase>> rules;
     if (isNestedContext()) {
         runInNewNestingContext([&]() {
-            consumeStyleBlock(block, StyleRuleType::Style);
+            static const auto isParsingStyleDeclarationsInRuleList = true;
```

Comment:
What is the goal of this const variable instead of directly passing `true` to the consumeStyleBlock function ?

Reply:
maybe have an `enum class ParsingStyleDeclarationsInRuleList : bool { No, Yes }` instead?